etc/icons.example

```diff
@@ -359,3 +375,212 @@ Vagrantfile             
 
 # other formats
 *.pdf   
+
+# icons with color
```

Comment:
OPTIONAL: This is just an idea, maybe it might be worth moving this into a separate file (e.g. `icons_colored.example`). But either way is OK for me.

Reply:
I think that is a good idea too. Already moved.